src/Data/Vec/Relation/Binary/Equality/Setoid.agda

```diff
@@ -79,6 +80,12 @@ map-++ : ∀ {b m n} {B : Set b}
 map-++ f []       = ≋-refl
 map-++ f (x ∷ xs) = refl ∷ map-++ f xs
 
+map-[]≔ : ∀ {b n} {B : Set b}
+  (f : B → A) (xs : Vec B n) i p →
+  map f (xs [ i ]≔ p) ≋ map f xs [ i ]≔ f p
```

Comment:
Can you align these as per the style guide (i.e in the same manner as the other function)?

Reply:
It is with the same indentation as `map-++`.

Reply:
Err something is still wrong with the indentation. Are you using tabs or non-standard number of spaces in your editor?

Reply:
Maybe are you looking at this outdated version?

I looked at this file using VSCode and it is just using whitespace, without tabs.
Also, looking at Github, it is the same that expect it to be.

Another possibility is that maybe I think it is idented right when it isn't.